status/types.proto

```diff
@@ -5,7 +5,7 @@ package neo.fs.v2.status;
 option go_package = "github.com/nspcc-dev/neofs-api-go/v2/status/grpc;status";
 option csharp_namespace = "Neo.FileStorage.API.Status";
 
-// Declares the general format of the status returns of the NeoFS RPC protocol.
+// It declares the general format of the status returns of the NeoFS RPC protocol.
```

Comment:
Let's use just the action verb in the beginning of a method description. It's a common practice, I think.